### DISCUSSION TITLE
Glb file is not loading in mobile

### DISCUSSION THREAD

**Discussion by goyalsunil452** [google/model-viewer#4470] [Q&A] [answered]
### Description
When I open the same url in desktop browser I am able to see the loaded GLB (3d file) and I hosted my application on ip address to check in mobile (ios and android) and glb file is not getting loaded.
Not able to figure out the root cause.

#### Live Demo

https://glitch.com/edit/#!/model-viewer

In Mobile : https://drive.google.com/file/d/1LC9yOodA9vbtHCkdd3ylN2F1jEADXWXN/view?usp=sharing
In desktop : https://drive.google.com/file/d/1r1sslCS_NRQQSXHCMDppTfB06CnthcTy/view?usp=sharing

This is the URL where file is stored : https://carnivalsb.nslhub.com/dsd-orch/dsd-files-store/content/cduiprod/other/american_muscle_car4f401d22-7f2c-48c7-bbbe-61ab8f8d5a89.glb
### Version

- model-viewer: v3.1.1

### Browser Affected

- [x] Chrome, version:116.0.5845.190
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

### OS

- [ ] Android 
- [ ] iOS 15 +
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### AR

- [ ] WebXR
- [ ] SceneViewer  <--
- [ ] QuickLook

**Comment by elalish** [comment]
Have you checked the JS console for errors/warnings? Try chrome://inspect Most likely issue is CORS headers.

**Comment by goyalsunil452** [comment]
Hello, @elalish, Thanks for the reply.
I am using a Mobile (iOS device ) to run this application so, I can not inspect it.
On the desktop web it is working fine, so there are no console errors

**Reply by elalish** [reply, marked as answer]
On iOS you can inspect it by plugging in to a mac, opening desktop Safari and going into the Developer menu.

**Reply by goyalsunil452** [reply]
Found the issue.
In the mobile, URL( https://carnivalsb.nslhub.com/dsd-orch/dsd-files-store/content/cduiprod/other/american_muscle_car4f401d22-7f2c-48c7-bbbe-61ab8f8d5a89.glb) is giving error as not able to fetch the data. because it is authenticated with some private key.
![IMG_0950](https://github.com/google/model-viewer/assets/79658759/48c69108-0049-49ca-8afb-85da3c4a7ed0)

Thanks for your time @elalish.
